docs/react-wiki-archive/References/Browser-Support.md

```diff
@@ -0,0 +1,52 @@
+# Browser support
```

Comment:
Potential Keeper

Reply:
Does this conflict with our v9 story book? Check with Tristan.

Relevant link: https://react.fluentui.dev/?path=/docs/concepts-developer-browser-support-matrix--page

Reply:
Conversation initiated with Tristan via Teams.

Reply:
Outcome of conversation with Tristan:

We should only have 1 source of truth for our browser support matrix: the v9 storybook. We'll drop this document.